tests/nodeUnit.js

```diff
@@ -22,10 +22,60 @@ define(function(require) {
   const bdd = require('intern!bdd');
   const assert = require('intern/chai!assert');
   const fs = require('intern/dojo/node!fs');
+  const path = require('intern/dojo/node!path');
 
   const publicDir = 'dist/public';
 
+  function rm(filepath) {
```

Comment:
You can use `del` instead of rolling your own.

https://www.npmjs.com/package/del